http/mixed-content.json

```diff
@@ -0,0 +1,322 @@
+        }
+      },
+      "auto_upgrade_images": {
+        "__compat": {
+          "description": "Upgradable image mixed content by default",
+          "spec_url": "https://www.w3.org/TR/mixed-content/#intro",
+          "support": {
+            "chrome": {
+              "version_added": "86"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "127",
+              "notes": [
+                "Set <code>security.mixed_content.upgrade_display_content</code> preference to <code>true</code> to allow HTTP fetching and display of upgradable content",
+                "Set <code>security.mixed_content.block_display_content</code> preference to <code>true</code> to block all mixed content"
+              ]
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "auto_upgrade_video_audio": {
+        "__compat": {
+          "description": "Upgrade video and audio content by default",
+          "spec_url": "https://www.w3.org/TR/mixed-content/#intro",
+          "support": {
+            "chrome": {
+              "version_added": "80"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "127",
+              "notes": [
+                "Set <code>security.mixed_content.upgrade_display_content</code> preference to <code>true</code> to allow HTTP fetching and display of upgradable content",
+                "Set <code>security.mixed_content.block_display_content</code> preference to <code>true</code> to block all mixed content"
+              ]
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "block_active_content": {
+        "__compat": {
+          "description": "Block active mixed content (later 'blockable' mixed content).",
```

Comment:
Wasn't sure what to call this. Active content is the content that has always been blocked because it could modify the document, and "passive" or "optionally blockable" was the stuff that couldn't. Now the spec has upgradable and blockable content. The upgradable stuff is images, video, audio - and likely to either stay that or be reduced. The blockable stuff is everything else.

So blockable and active completely overlap now, but in future there will be new types that are blockable but not active. I have called it active and fudged the difference. THoughts?